src/coreclr/src/vm/method.cpp

```diff
@@ -83,15 +83,32 @@ static_assert_no_msg((sizeof(DynamicMethodDesc)     & MethodDesc::ALIGNMENT_MASK
     adjustment + sizeof(ComPlusCallMethodDesc),      /* mcComInterOp    */  \
     adjustment + sizeof(DynamicMethodDesc)           /* mcDynamic       */
 
-const SIZE_T MethodDesc::s_ClassificationSizeTable[] = {
+const BYTE MethodDesc::s_ClassificationSizeTable[] = {
```

Comment:
should this change match the assignment at line 142 below?

Reply:
No, its not necessary. All of these sizes are well under 1 byte in length, and there is no reason to have a big table. (Zero-extending bytes to size_t is extremely cheap/free.)